.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
Please, remove and use a global gitignore file on your laptop.

https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer